databuilder/databuilder/models/feature/feature_watermark.py

```diff
@@ -18,6 +18,8 @@ class FeatureWatermark(GraphSerializable):
     """
     NODE_LABEL = 'Feature_Watermark'
 
+    TYPE_ATTR = 'watermark_type'
```

Comment:
this is unrelated to the `Usage` stuff. Just a minor improvement in the ongoing feature work.